docs/SuperSIDonRaspi.md

```diff
@@ -25,60 +43,87 @@ Get the source from GitHub.com
+other sources such as other SuperSID users, and Xray flux data collected by
+satellites.
+If your machine does not have ntp installed then you want to install it:
 ```console
     $ sudo apt-get install ntpdate ntp
 ```
-Follow the tutorial [Raspberry Pi sync date and time](https://victorhurdugaci.com/raspberry-pi-sync-date-and-time)
 
-Optional: Virtualenv management for Python:
+Follow the tutorial [Raspberry Pi sync date and time]
+(https://victorhurdugaci.com/raspberry-pi-sync-date-and-time)
+
+Optional: Virtual environment management for Python:
+If your machine is being used for other purposes as well as SuperSID, you may
+want to install SuperSID in its own virtual python environment. This way you
+will not be plagued with version conflicts with the various python packages
+installed for different uses. There are several different ways to set up a
+virtual environment. This is one that has worked for me.
+
+
+https://docs.python.org/3/tutorial/venv.html
+
+Install the venv package.
 ```console
-    $ sudo apt-get install mkvirtualenv
+    $ sudo python3 -m pip install venv
 ```
 
 ## 3) Installing SuperSID
 
 ### 3.1) optional virtual environment
 
-This step is optional. Creating your own environment allows to install libraries in all freedom,
-without `sudo` and ensure you have a coherent and working set of libraries (sound card).
-If your Raspi is dedicated to SuperSID then you can skip this step and install all globally.
+This step is optional. Creating your own environment allows to install libraries
+in all freedom, without `sudo` and ensure you have a coherent and working set of
+libraries (sound card). If your Raspi is dedicated to SuperSID then you can skip
+this step and install all globally.
 
 From /home/pi:
 ```console
     $ cd ~/supersid
-    $ mkvirtualenv -p /usr/bin/python3 supersid
-    $ workon supersid
-    $ toggleglobalsitepackages
+    # Create your virtual environement
+    $ python3 -m venv supersid-env
+    # Activate the newly created virtual environment
+    $ source supersid-env/bin/activate
+    # Install latest versions of package tools
+    $ python -m pip install -U pip wheel setuptools
 ```
 
-Your prompt should now start with '(supersid)'
-
-This also ensures that we run in Python 3.7.3 as per current configuration.
-
+Your prompt should now start with '(supersid-env)'
+This ensures that we run in Python 3.x as per current configuration.
 
 ### 3.2) Global or local installation
 
-This Raspi 3 is dedicated to SuperSid or you do not plan to mix various libraries: install at system level all the libraries.
-You can do so exactly like you would do in linux, for an local installation inside the virtual environement by first executing 'workon supersid'.
+This Raspberry Pi is dedicated to SuperSid or you do not plan to mix various
+libraries: install at system level all the libraries.
 
+For an local installation inside the virtual environment, first execute 'workon
```

Comment:
See earlier comment about workon

Reply:
Copied from your version of docs/SuperSIDonLinux.md.

Again, let me know which wrong text to be replaced with which correct text.

Reply:
Would this cahnge be correct?

old
> For an local installation inside the virtual environment, first execute 'workon supersid-env'.

new
> For an local installation inside the virtual environment, first execute 'source supersid-env/bin/activate'.

or more generic (the console commands follow immediately):
> For an local installation inside the virtual environment, first activate the virtual environment.

Reply:
I'd go with first execute 'source supersid-env/bin/activate'.